tests/runtime/bool

```diff
@@ -1,5 +1,21 @@
+NAME bool as map keys and values
+PROG begin { @a[true] = false;  }
```

Comment:
Does this test the default printing or why is it not a self test? Same question for the other new tests in this suite.